curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/remove-items-using-splice.english.md

```diff
@@ -31,19 +31,24 @@ let newArray = array.splice(3, 2);
 
 ## Instructions
 <section id='instructions'>
-We've defined a function, <code>sumOfTen</code>, which takes an array as an argument and returns the sum of that array's elements. Modify the function, using <code>splice()</code>, so that it returns a value of <code>10</code>.
+
+We've initialized an array `arr`. Use `splice()` to remove elements `arr`, so that it only contains elements that sum to the value of <code>10</code>.
```

Comment:
```suggestion
We've initialized an array `arr`. Use `splice()` to remove elements from`arr`, so that it only contains elements that sum to the value of <code>10</code>.
```

Reply:
Thanks for catching that.